test/built-ins/Temporal/PlainYearMonth/compare/argument-propertybag-calendar-leap-second.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
I saw a few brand-new tests in this PR that seem unrelated to the nested property bags. That seems fine to me, just wanted to confirm that this was intended.

Reply:
Yeah, the story behind this one is that it should've been here but for some reason was missing from PlainYearMonth.compare. (Probably an oversight on my part.) It would've previously had a nested property bag in it, which I would've removed in this PR.